posts/2019/07/ibis-python-data-analysis-productivity-framework.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ibis: Python Data Analysis Productivity Framework\n",
```

Comment:
Can you remove this? It's duplicate with the title

Reply:
Also a minor thing about the title, could you change the capitalization to "Ibis: Python data analysis productivity framework" ?

Reply:
sure. I am doing that. thanks!